CMakeLists.txt

```diff
@@ -672,11 +672,11 @@ if (BUILD_TESTS)
     )
 
     # gtest
-    add_library(gtest STATIC
+    add_library(gtest
       third_party/gtest/googletest/src/gtest-all.cc
     )
 
-    add_library(gtest_main STATIC
+    add_library(gtest_main
```

Comment:
Same with these test-only libraries, lets just leave them as static for now.  (and the fuzzer one).